app/build.gradle

```diff
@@ -297,6 +297,9 @@ dependencies {
     coreLibraryDesugaring "com.android.tools:desugar_jdk_libs:$versions.desugar_jdk_libs"
 
     profileImplementation "com.squareup.leakcanary:leakcanary-android:$versions.leak_canary"
+
+    // Unstoppabledomains
+    implementation 'com.unstoppabledomains:resolution:1.12.1'
```

Comment:
please add version to versions.gradle file and convert this to 
```
implementation 'com.unstoppabledomains:resolution:$versions.unstoppabledomains'
```

Reply:
done